nyc/bills.py

```diff
@@ -161,133 +161,150 @@ def sponsorships(self, matter_id):
+                              classification=action['classification'])
 
+        result, votes = vote
 
-            if text:
-                if text['MatterTextPlain']:
-                    bill.extras['plain_text'] = text['MatterTextPlain'].replace(u'\u0000', '')
+        if result:
+            organization = json.loads(act['organization_id'].lstrip('~'))
+            vote_event = VoteEvent(legislative_session=bill.legislative_session,
+                                   motion_text=act['description'],
+                                   organization=organization,
+                                   classification=None,
+                                   start_date=act['date'],
+                                   result=result,
+                                   bill=bill)
 
-                if text['MatterTextRtf']:
-                    bill.extras['rtf_text'] = text['MatterTextRtf'].replace(u'\u0000', '')
+            legistar_web, legistar_api = [src['url'] for src in bill.sources]
 
+            vote_event.add_source(legistar_web)
+            vote_event.add_source(legistar_api + '/histories')
 
-            for action, vote in self.actions(matter_id):
-                act = bill.add_action(action['action_description'],
-                                      action['action_date'],
-                                      organization={'name': action['responsible_org']},
-                                      classification=action['classification'])
+            for vote in votes:
+                raw_option = vote['VoteValueName'].lower()
 
-                result, votes = vote
+                if raw_option == 'suspended':
+                    return
 
-                if result:
-                    organization = json.loads(act['organization_id'].lstrip('~'))
-                    vote_event = VoteEvent(legislative_session=bill.legislative_session,
-                                           motion_text=act['description'],
-                                           organization=organization,
-                                           classification=None,
-                                           start_date=act['date'],
-                                           result=result,
-                                           bill=bill)
+                clean_option = self.VOTE_OPTIONS.get(raw_option, raw_option)
+                vote_event.vote(clean_option, vote['VotePersonName'].strip())
 
-                    vote_event.add_source(legistar_web)
-                    vote_event.add_source(legistar_api + '/histories')
+            return vote_event
 
-                    for vote in votes:
-                        raw_option = vote['VoteValueName'].lower()
 
-                        if raw_option == 'suspended':
-                            continue
+    def scrape(self, window=3, matter_id=None):
+        self.version_errors = []
+
+        if matter_id:
+            matters = [self.fetch(matter_id)]
```

Comment:
I think `matter` would be a clearer name.

Reply:
Is `matters` ok given [this change](https://github.com/opencivicdata/scrapers-us-municipal/pull/182/commits/c52cca469f2babcd5509068645ea5fc6d5b539cd#diff-0d0f0450f469b3ad1598867cde71cb7bR291)?